lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -116,6 +124,80 @@ function preflightCheck(
     }
 }
 
+/**
+ * Public method to indicate whether platform broker is available to make native token request.
+ */
+export async function isPlatformBrokerAvailable(
```

Comment:
Can we put this in a new file under the `broker` folder instead?

Reply:
Okay sure, I can just add it to the new Handler file that will handle DOM requests.